src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -724,10 +725,25 @@ export class PatientViewPageStore {
             );
         },
     });
+    @observable _selectedMutationalSignatureVersion: string;
+
+    readonly initialMutationalSignatureVersion = remoteData({
+        await: () => [],
+        invoke: () => {
+            return Promise.resolve(
+                retrieveMutationalSignatureVersionFromData(
+                    this.fetchAllMutationalSignatureData.result.map(
+                        profile => profile.molecularProfileId
+                    )
+                )
+            );
+        },
+        default: 'v2',
+        onResult: version => {
```

Comment:
The onResult could indeed be removed. However, I believe `default: 'v2` is needed. If we remove this, the test on the PatientViewPageTabs will return an error because  `string | undefined` is not allowed for:
```
version={
              pageComponent.patientViewPageStore.selectedMutationalSignatureVersion
}
```
The test passes when we set `default:v2`